tests/Command/ListCollectionsTest.php

```diff
@@ -1,9 +1,10 @@
 <?php
 
-namespace MongoDB\Tests\Operation;
+namespace MongoDB\Tests\Command;
```

Comment:
Since these were renamed, does that mean there is no longer a test for `MongoDB\Operation\ListCollections` and `ListDatabases`? I suppose that's not a problem since they really don't do much at all, and the operation functional tests remained in place.

I'll also note that in the case of FindAndModify and its derivative operations, the operation tests only deal with special options where validation logic is in the derivative class. There's nothing left in the ListCollections operation to actually test.

LGTM, just wanted to point this out.

Reply:
`ListCollectionsTest` only tests parameter parsing, which is why I moved the tests to the command namespace. Functional testing of the operations is done in `ListCollectionsFunctionalTest` in the `Operation` test namespace.